Saturation/SaturationAlgorithm.cpp

```diff
@@ -733,8 +725,8 @@ void SaturationAlgorithm::init()
     _symEl->init(this);
   }
 
-  _startTime = env.timer->elapsedMilliseconds();
-  _startInstrs = env.timer->elapsedMegaInstructions();
+  _startTime=Timer::elapsedMilliseconds();
+  _startInstrs=Timer::elapsedMegaInstructions();
```

Comment:
Maybe these could now go too?

Reply:
Oh, I didn't realized it's for LRS. 

Hmm, but while I see `_startTime` there, `_startInstrs` seem to be missing (although by analogy they should probably appear on line 113 there). What do you think?

Reply:
OK, so `_startInstrs` is definitely not used. Now that I look I'm not sure that what LRS does with `_startTime` makes sense either, probably you don't care when you started saturating so much as when the Vampire run started. Should we get rid of `_startTime` as well and just use the elapsed time for LRS, or do you have some counter-argument?

Reply:
I think it's important that we don't start LRS-ing before we've made some number of iterations around the saturation loop, but that's happening on the line `    if (processed<=10) {`. 

The `_startTime` and `_startInstrs` are there to implement `opt.lrsFirstTimeCheck()` and there it's probably even more intuitive to count the time from the start of things (i.e. not from the start of saturation). Moreover, the current solution for time is confusing: e.g., lrsFirstTimeCheck=25% and half of the time is spent parsing and preprocessing, then the first check will only kick in at around 75% allotted time (because it ignores startup, but only estimates the %time spent from the beginning). Long story - I think the can both go and we can implement `lrsFirstTimeCheck` along the lines of the instr version, which does not need `_startInstrs`.